File: packages/create-react-native-library/templates/java-view-mixed/android/src/main/java/com/{%- project.package_dir %}/{%- project.name %}View.java
package com.<%- project.package %>;

import androidx.annotation.Nullable;

import android.content.Context;
import android.util.AttributeSet;

import android.view.View;

public class <%- project.name -%>View extends View {

  public <%- project.name -%>View(Context context) {
    super(context);
  }

  public <%- project.name -%>View(Context context, @Nullable AttributeSet attrs) {
    super(context, attrs);
  }

  public <%- project.name -%>View(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
    super(context, attrs, defStyleAttr);
  }

}

File: packages/create-react-native-library/templates/kotlin-view-mixed/android/src/main/java/com/{%- project.package_dir %}/{%- project.name %}View.kt
package com.<%- project.package %>

import android.content.Context
import android.util.AttributeSet
import android.view.View

class <%- project.name -%>View : View {
  constructor(context: Context?) : super(context)
  constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
  constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
    context,
    attrs,
    defStyleAttr
  )
}
